io-engine/src/lvm/vg_pool.rs

```diff
@@ -149,6 +178,14 @@ impl VolumeGroup {
             .with_vg(CmnQueryArgs::ours().uuid(self.uuid()).named(self.name()));
         LogicalVolume::list(&query).await
     }
+    async fn list_foreign_lvs(&self) -> Result<Vec<LogicalVolume>, Error> {
```

Comment:
Might consider naming as `list_foreign_lvm_lvols` to avoid confusion with `lvs` backend. Same with `list_lvs`

Reply:
Here's how I think about this wrt our 2024 Roadmap thinking...
lvs = SPDK LVS, Lvol, Bdev backend
lvm = LVM backend
lzp = ZFS Zpool backend